src/App/App.js

```diff
@@ -7,49 +7,43 @@ import CheckOutForm from "../CheckOutForm/CheckOutForm";
 import { Route } from "react-router-dom";
 import { ApolloProvider } from '@apollo/react-hooks';
 import ApolloClient from 'apollo-boost';
+import { CurrentCenterContext } from '../CurrentCenterContext';
 const client = new ApolloClient({
   uri: 'https://safe-space-be.herokuapp.com/graphql',
 });
 
-class App extends Component {
-  constructor() {
-    super();
-    this.state = {
-      reliefCenters: [],
-      reliefCenterID: "4",
-      isLoading: false
-    };
-  }
+const App = () => {
+  const [reliefCenter, setreliefCenter] = useState('');
+  const [isCenterSelected, selectCenter] = useState(false)
+  const [isLoading, setIsLoading] = useState(false);
 
-  selectPin = () => {
-    console.log('test')
-  }
-
-  //add componentDidMount to fetch relief center data
-  // once fetched, set isLoading to false
-
-  render() {
+  const currentReliefCenter = useMemo(() => ({ reliefCenter, setreliefCenter }), [reliefCenter, setreliefCenter]);
+    
+    //add componentDidMount to fetch relief center data
+    // once fetched, set isLoading to false
```

Comment:
Let's get rid of these as we work through the next PR; I'm guessing that since our fetch lives in the Map component, we may not need to worry about mounting anything here.